.github/workflows/cache.yaml

```diff
@@ -0,0 +1,30 @@
+name: cache
+on: workflow_call
+concurrency:
+  group: cache-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
@matthieu-foucault even in the context of moving the concurrency group up to `main.yaml` this workflow will always be manually called so I figured we'd leave this here?

Reply:
...or maybe not since I'm proposing we trigger this on schedule with nightlies. If we land the concurrency refactor first I'll address?

Reply:
Not sure we should worry about this workflow's concurrency. We can probably leave this out?

Reply:
Conceivably could take a while to iterate through the loop if it was a busy day with a large cache and I didn't want a race condition where new job is also iterating in parallel